protocol/messenger.go

```diff
@@ -1775,221 +1781,6 @@ func (m *Messenger) handlePushNotificationClientRegistrations(c chan struct{}) {
 	}()
 }
 
-// InitFilters analyzes chats and contacts in order to setup filters
-// which are responsible for retrieving messages.
-func (m *Messenger) InitFilters() error {
```

Comment:
I agree it would facilitate review to follow your suggestion @igor-sirotin, but would simply rewriting the history with separate commits and keep one PR be enough since this PR is relatively simple if we consider a good chunk of the code is identical to develop?

Usually my personal approach is to have a separate commit for file moves and renames in an attempt to let git understand what's going on. Then separate commits for each refactor technique applied.

Reply:
Yeah, separate commits works as well 👍